src/components/v5/common/Fields/InputBase/FormInputBase.tsx

```diff
@@ -47,7 +48,14 @@ const FormInputBase: FC<FormInputBaseProps> = ({
         if (type === 'number') {
           onChange(Number.isNaN(valueAsNumber) ? 0 : valueAsNumber);
         } else {
-          onChange(inputValue);
+          onChange(
```

Comment:
i don't really understand this, we have an input that is not of type `number` but it can allow only numbers?
I feel like we should be using type="number" and maybe combine it with `cleave`?